### PR TITLE
feat(i18n): views-vcs

### PR DESCRIPTION
**Pull request by Jose-Matsuda**
Implement i18n for classes under workbench.views.vcs

### Intent

This PR tackles hard-coded strings to i18n-ify in org.rstudio.studio.client.workbench.views.vcs (excluding false-positives).
Issue: https://github.com/StatCan/daaas/issues/732

### Approach

This pull-request and future ones will tackle i18nify classes at package-level (e.g. org.rstudio.core) instead of the per-component approach (like we did for all-options-menu)

Implementation follows the same pattern from #9485 as described in gwt/tools/i18n-helpers/README.md. Running the create_dev_locale.sh will create all the new "dev" locale from any new "en" properties files for easier visual confirmation. Refer to README for more information.

### Automated Tests

> Indicate whether this change includes unit tests or integration tests, or link a work item or pull request to add those tests to another repo. If the change cannot or will not be covered by a test, indicate why.

### QA Notes

> Add additional information for QA on how to validate the change, paying special attention to the level of risk, adjacent areas that could be affected by the change, and any important contextual information not present in the linked issues. 

Modify an entry in one of the new _en.properties files, and refresh RStudio with ant in devmode.
Run create_dev_locale.sh to clone all _en properties files, which will add a prefix of "@" to each constant and message. Then open RStudio using http://localhost:8787/?locale=dev and browse to the Options->* menus. You can also view in desktop mode by swapping the locale element value attribute from value="en" to value="dev" in RStudio.gwt.xml.

### Checklist

- [ ] If this PR adds a new feature, or fixes a bug in a previously released version, it includes an entry in `NEWS.md` 
- [x] If this PR adds or changes UI, the updated UI meets [accessibility standards](https://github.com/rstudio/rstudio/wiki/Accessibility)
- [ ] A reviewer is assigned to this PR (if unsure who to assign, check Area Owners list)
- [ ] This PR passes all local unit tests
- [x] Signed the contributor agreement

cc @gtritchie 

### NOTE
Left **4 TODO** comments regarding some things I were unsure about internationalizing or not

### CODE REVIEW

File: src/gwt/src/org/rstudio/studio/client/workbench/views/vcs/common/diff/LineTableView.java [hunk 1]
```diff
@@ -148,8 +149,8 @@ public void render(Context context, ChunkOrLine value, SafeHtmlBuilder sb)
                      sb,
                      RES.cellTableStyle().lineActions(),
                      selectionModel_.getSelectedSet().size() > 1
-                     ? " selection"
-                     : " line");
+                     ? constants_.selectionSuffix("")
```
Comment (gtritchie) [inline src/gwt/src/org/rstudio/studio/client/workbench/views/vcs/common/diff/LineTableView.java:152]: Is this technique (passing an empty string to placeholder) required to have a leading space at the start of the localizable string?

`selectionSuffix={0} selection`

Guessing you can't do this?

`selectionSuffix= selection`

File: src/gwt/src/org/rstudio/studio/client/workbench/views/vcs/common/diff/UnifiedParser.java [hunk 2]
```diff
@@ -187,6 +187,7 @@ else if (directive != StringUtil.charAt(diffLine,i))
 
    private boolean isNewFileLine(String nextLine)
    {
+      //TODO unsure about this check with Gary. I did change some SizeWarningWidgets to use `constants_`
```
Comment (gtritchie) [inline src/gwt/src/org/rstudio/studio/client/workbench/views/vcs/common/diff/UnifiedParser.java:190]: Do not localize these. They are for parsing output of git commands, and not shown in the UI.

File: src/gwt/src/org/rstudio/studio/client/workbench/views/vcs/git/GitStatusRenderer.java [hunk 3]
```diff
@@ -98,6 +98,7 @@ private String descForStatus(char c)
    {
       switch (c)
       {
+         // TODO Unsure about these 'case's, might need to leave be for the time
```
Comment (gtritchie) [inline src/gwt/src/org/rstudio/studio/client/workbench/views/vcs/git/GitStatusRenderer.java:101]: You can localize the return strings here. Don't localize the case values. These strings show up in the Git UI if you hover the mouse over the status graphic (e.g. [M] in screenshot below).

Lower in this file, the `imgForStatus()` function is returning those graphics which contain a letter, such as the [M]. We aren't localizing those graphics, but having the tooltip localized would be good.

Screen Shot 2021-12-08 at 16 52 06

File: src/gwt/src/org/rstudio/studio/client/workbench/views/vcs/svn/SVNStatusRenderer.java [hunk 4]
```diff
@@ -92,6 +92,7 @@ private String descForStatus(String str)
       
       switch (c)
       {
+         // TODO Similar to GitStatusRender unsure about cases
```
Comment (gtritchie) [inline src/gwt/src/org/rstudio/studio/client/workbench/views/vcs/svn/SVNStatusRenderer.java:95]: Same answer as for GitStatusRenderer. Localize the returns strings, but not the case values. These strings show up as tooltips in the Subversion UI.

I'd show a screenshot but then I'd have to remember how to setup svn.

**Review by gtritchie — CHANGES_REQUESTED** [review]

File: src/gwt/src/org/rstudio/studio/client/workbench/views/vcs/common/diff/LineTableView.java (hunk 1 above)
Comment (Jose-Matsuda) [reply src/gwt/src/org/rstudio/studio/client/workbench/views/vcs/common/diff/LineTableView.java:152]: Yup that's correct, doing `selectionSuffix= selection` will just have `selectionSuffix` resolving to `selection` with no leading space. If you would prefer, we could change it to be `" " + constants_.selectionSuffix()`.

Comment (gtritchie) [reply src/gwt/src/org/rstudio/studio/client/workbench/views/vcs/common/diff/LineTableView.java:152]: It's fine as-is. I've not worked with Java property files before so wasn't sure.

**Review by gtritchie — APPROVED** [review]

**Pull request merged**